middleware.go

```diff
@@ -27,19 +30,30 @@ func ClaimsToContextMiddleware(handler http.Handler, header string, idpKey inter
 
 // RequireClaim checks if the requests claims contain a specific value for a specific key
 func RequireClaim(handler http.Handler, claimKey, expectedClaimValue string) http.Handler {
+	log := logrus.WithField("require-claim", claimKey)
+	log = log.WithField("expected-value", expectedClaimValue)
```

Comment:
I'd prefer to see:
```go
logrus.
  WithField("require-claim", claimKey).
  WithField("expected-value", expectedClaimValue)
```

But I don't have a strong opinion on this :)

Reply:
I agree with this